tcb/.github/workflows/build-torizoncore.yaml

```diff
@@ -18,6 +18,15 @@ jobs:
         run: |
           echo "${{ github.workspace }}" >> abs-path
           cat abs-path
+      - name: Install build tools in container
+        run: | 
+          apt-get update
+          apt-get install -y wget curl libicu-dev
+          curl -fsSL https://get.docker.com -o get-docker.sh 
+          sh get-docker.sh
+                    
+          wget -nv https://github.com/PowerShell/PowerShell/releases/download/v7.4.2/powershell_7.4.2-1.deb_amd64.deb
+          dpkg -i powershell_7.4.2-1.deb_amd64.deb
```

Comment:
this is needed because I'm running the build inside a container, on the shared and self-hosted runners. Both will spin up a docker Ubuntu:23.04 container, mount the local filesystem, and run the build commands. It's good for reproducibility. If you used GitLab, running dind is standard.